cmd/internal/planetscale_edge_database.go

```diff
@@ -3,6 +3,7 @@ package internal
 import (
 	"context"
 	"fmt"
+	"golang.org/x/exp/slices"
```

Comment:
I think we should upgrade to Go 1.21 and then `slices` can be used instead of this package.

Reply:
yeah I agree! will update once your Go-upgrade PR gets merged